src/_includes/subnav.html

```diff
@@ -0,0 +1,16 @@
+{% if page.next or page.prev -%}
+<nav id="subnav">
+  <ul>
+    <li class="previous">
+      {% if page.prev.path -%}
```

Comment:
can we move the if statement to wrap the <li> so that it won't output anything when there's no previous or next?

Reply:
I've fixed the styling issues and now I can move the `<li>` inside the if.